setup.cfg

```diff
@@ -7,22 +7,22 @@ test             = pytest --addopts "-n auto -m 'not dockertest'"
 testall          = pytest --addopts "--cov-report=xml --cov=gordo_components"
 
 # Only run tests which use docker
-testdocker       = pytest --addopts "-m 'dockertest'"
+testdocker       = pytest --addopts "-m 'dockertest' --cov-report=xml --cov=gordo_components"
 
 # Component level filtering of tests
-testbuilder      = pytest --addopts "tests/gordo_components/builder"
-testcli          = pytest --addopts "tests/gordo_components/cli"
-testclient       = pytest --addopts "tests/gordo_components/client"
-testdataprovider = pytest --addopts "tests/gordo_components/data_provider"
-testdataset      = pytest --addopts "tests/gordo_components/dataset"
-testmodel        = pytest --addopts "tests/gordo_components/model"
-testserializer   = pytest --addopts "tests/gordo_components/serializer"
-testserver       = pytest --addopts "tests/gordo_components/server"
-testutil         = pytest --addopts "tests/gordo_components/util"
-testwatchman     = pytest --addopts "tests/gordo_components/watchman"
+testbuilder      = pytest --addopts "tests/gordo_components/builder --cov-report=xml --cov=gordo_components"
```

Comment:
I _think_ you would only need to at this to the `pytest.ini` where we set a bunch of the default opts.